packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -185,6 +190,14 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                         }),
                     text: translationString('TR_CONVERT_TO_LOWERCASE'),
                 };
+            case 'contract':
+                return {
+                    onClick: () => {
+                        contractAddressWarningDismissed = true;
```

Comment:
so this should work just once per send form session?

Reply:
Hm, interesting, but that's not clean code. Could you please come up with a solution which will also replace
```
    const [hasAddressChecksummed, setHasAddressChecksummed] = useState<boolean | undefined>();
```
there can be one state for both `evmChecks` (object)

Reply:
I think that due to the nature of the errors it would be better to later group them into objects based on their logic

These 2 errors require different closure, checksum is dependent on the actual state of the address and can't be affected by the user's behavior, also when the address is changed I want to display bottom text and it works well with useState variable. 

Contract on the other side requires an action from the user and can be dismissed and I don't want it to be shown only based on the state of the address (the address can be the same but state of the error can be different)

So I don't want to put these errors together in an object because it requires picking either state based handling or normal variable, and the other error logic breaks 

This is why I would group some other errors later based on their logic

If I've missed something or misunderstood your request please do let me know, but this is my conclusion after approx. a day playing with the logic of these errors

Reply:
Okay, makes sense. However, I still don't like "let" variable here and be dependant on rerender

Reply:
@tomasklim I found a way to achieve the same behavior by using useRef, it's in the fixup